classes/Search.php

```diff
@@ -23,7 +23,14 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
+
 define('PS_SEARCH_MAX_WORD_LENGTH', 15);
+define('PS_SEARCH_LEVENSHTEIN_LOOP', 10); /* Max time Levenshtein algo is use by search*/
```

Comment:
These const must be configurable, maybe in adavanced menu. Or in a new Pull request?